core-bundle/tests/Doctrine/Backup/BackupManagerTest.php

```diff
@@ -428,8 +428,13 @@ private function getBackupDir(): string
 
     private function getBackupManager(Connection $connection = null, DumperInterface $dumper = null): BackupManager
     {
-        $connection = $connection ?? $this->createMock(Connection::class);
-        $dumper = $dumper ?? $this->createMock(DumperInterface::class);
+        if (null === $connection) {
+            $connection = $this->createMock(Connection::class);
+        }
+
+        if (null === $dumper) {
+            $dumper = $this->createMock(DumperInterface::class);
+        }
```

Comment:
```suggestion
        $connection ??= $this->createMock(Connection::class);
        $dumper ??= $this->createMock(DumperInterface::class);
```

Reply:
This applies to all code blocks of this kind in this PR I think.

Reply:
Why is this change necessary in the first place though? I mean what's the problem with the original version?

Reply:
This is the original problem:

<img width="1159" alt="" src="https://user-images.githubusercontent.com/1192057/144633764-357ff736-2ab2-4fd2-849e-df3eac6aba39.png">

But @ausi's suggestion to use `??=` solves the problem.

Reply:
Changed in f2c019aec55bade213d1f2dc37f079a4a1a031e6.